maintainer/travis/build_cmake.sh

```diff
@@ -74,17 +74,21 @@ fi
 # enforce style rules
 grep 'class[^_].*[^\)]\s*:\s*$' $(find . -name '*.py*') && echo -e "\nOld-style classes found.\nPlease convert to new-style:\nclass C: => class C(object):\n" && exit 1
 
-if [ ! $insource ]; then
+if ! $insource; then
     if [ ! -d $builddir ]; then
         echo "Creating $builddir..."
         mkdir -p $builddir
     fi
 fi
 
-if [ ! $insource ]; then
+if ! $insource; then
     cd $builddir
 fi
 
+# load MPI module if necessary
+grep -q suse /etc/os-release && source /etc/profile.d/modules.sh && module load gnu-openmpi
+grep -q rhel /etc/os-release && source /etc/profile.d/modules.sh && module load mpi
```

Comment:
Manual `source`ing of the profile seems to be necessary because `docker run` does not produce a login shell.